projects/demo/src/modules/components/abstract/number-format-documentation/number-format-documentation.template.html

```diff
@@ -0,0 +1,68 @@
+<h6 class="tui-text_h6">
+    Can be expanded with
+    <a
+        routerLink="/directives/number-format"
+        target="_blank"
+        tuiLink
+    >
+        TuiNumberFormat
+    </a>
+</h6>
+<p>
+    Usage example:
+    <code>
+        [tuiNumberFormat]='&#123;decimal, precision, rounding, zeroPadding, thousandSeparator, decimalSeparator&#125;'
+    </code>
+</p>
+<p>
+    Requires you to import
+    <code>TuiNumberFormatModule</code>
+    .
+</p>
+
+<tui-doc-documentation>
+    <ng-template
+        documentationPropertyMode="input"
+        documentationPropertyName="decimal"
+        documentationPropertyType="TuiDecimal"
+        [documentationPropertyValues]="documentedComponent.decimalVariants"
+        [(documentationPropertyValue)]="documentedComponent.decimal"
+    >
+        Show/hide decimal
```

Comment:
Add this description, please

* `always` – number of digits after `decimalSeparator` is always equal to the `precision`.
* `pad` – pads trailing zeroes up to precision **on blur**
* `non-zero` – drops trailing zeroes up to precision **on blur**

Reply:
added, but without `on Blur`, for non-input components/utilities